docs/data/material/components/button-group/GroupOrientation.js

```diff
@@ -9,6 +9,8 @@ const buttons = [
   <Button key="three">Three</Button>,
 ];
 
+const ariaLabel = 'Vertical button group';
```

Comment:
I would inline it instead of extracting it in a const, it would be easier to copy the usage

Reply:
Oh, that makes sense!